manage_user_reset.php

```diff
@@ -56,20 +56,12 @@
 
 user_ensure_exists( $f_user_id );
 
-$t_user = user_get_row( $f_user_id );
+$t_data = array(
```

Comment:
Remove the access check for manager user threshold and user ensure exist checks above.  Remove require statements that are not longer necessary.